asab/library/providers/zookeeper.py

```
@@ -264,109 +262,179 @@ def _check_version_counter(self, version):
+							authz = Authz.get()
+							cred_id = getattr(authz, "CredentialsId", None)
+						except LookupError:
+							cred_id = None
+						node_path = self._personal_node_path(fname, tenant_id, cred_id) or ""
+					else:
+						node_path = self.build_path(fname, tenant_specific=(target == "tenant"))
+					zstat = await self.Zookeeper.exists(node_path) if node_path else None
 					size = zstat.dataLength if zstat else 0
 				except kazoo.exceptions.NoNodeError:
 					size = None
 				except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)